src/api.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @author acvetkov@yandex-team.ru
+ * @overview
+ */
+
+'use strict';
```

Comment:
You can remove `use strict` from ES6 modules. It's because of strict in modules by default